dask/dataframe/io/parquet.py

```diff
@@ -677,6 +677,13 @@ def _get_pyarrow_divisions(pa_pieces, divisions_name, pa_schema):
                 divisions_ns = [_to_ns(d, time_unit) for d in
                                 divisions]
                 divisions = [pd.Timestamp(ns) for ns in divisions_ns]
+
+            # Handle encoding of bytes string
+            if index_field.type == pa.string():
+                # Is this always true? Is the string encoding specified in the parquet metadata somewhere?
+                encoding = 'utf-8'
```

Comment:
All parquet strings are UTF8, there is no option to set any other encoding.

Reply:
Great! I'll remove the question and update the comment